Automattic/jetpack
# Search: Adjust z-index on customberg overlay to fix colorpicker popover display

Fixes #22713 

#### Changes proposed in this Pull Request:

* This PR fixes the colorpicker overlay issue, where part of the colorpicker popover is being cut off behind the instant search overlay due to the z-index value of the `.jetpack-instant-search` class.

#### Jetpack product discussion

* discussed in the comments of issue #22713 

#### Does this pull request change what data or activity we track or use?

no

#### Testing instructions:

* Go to the "Search configuration" page: [example.blog/wp-admin/admin.php?page=jetpack-search-configure](https://example.blog/wp-admin/admin.php?page=jetpack-search-configure)
* Try to change the highlight color
* Check that the entire colorpicker is visible, and is not being hidden behind the instant search overlay

| Before  | After |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/30754158/153939859-f44ebf73-6fad-47ea-9d69-bf525d431962.png)  | ![image](https://user-images.githubusercontent.com/30754158/153941044-46e495ec-c9ce-4f33-a10c-0dbe0d96e886.png)  |

Question: I have gone ahead and added it to the `.jetpack-instant-search` class directly. Is this okay, or would it be preferable to build it into `$z-layers` in the `_base-styles.css` file?

Additional note: currently, the colorpicker still lies underneath `components-panel__header` which I believe is how it should be. but feel free to correct me if we want it overtop this as well. 
<img src="https://user-images.githubusercontent.com/30754158/153939961-d943d912-e2e6-4fac-88cb-81ee8ed6f4d7.png" width="400" height="auto">
